app/src/main/java/ca/klapstein/baudit/models/DataModel.java

```diff
@@ -47,14 +51,19 @@ static public boolean uniqueID(String username) {
      * <p>
      * TODO: implement
      *
+     * considering param type {@code String}
      * @param username {@code String}
      * @param password {@code String}
      * @return {@code boolean}
      */
-    public boolean validateLogin(String username, String password) {
-        // TODO: implement from RemoteModel
+    public Patient validateLogin(/*String type, */String username, String password) {
```

Comment:
Why return `Patient` don't we also have to validate the login for a `CareProvider`?

Using a `boolean` makes more sense as this is a simple logic check. Not a getter method.